code/client/clrcore/External/VehicleHash.cs

```diff
@@ -9,19 +9,36 @@ public enum VehicleHash : uint
 		Adder = 3078201489u,
 		Airbus = 1283517198u,
 		Airtug = 1560980623u,
+		Akula = 1181327175u,
 		Akuma = 1672195559u,
+		Alkonost = 3929093893u,
 		Alpha = 767087018u,
+		AlphaZ1 = 2771347558u,
 		Ambulance = 1171614426u,
 		Annihilator = 837858166u,
+		Annihilator2 = 295054921u,
 		Apc = 562680400u,
+		ArbiterGT = 1549009676u,
 		Ardent = 159274291u,
 		ArmyTanker = 3087536137u,
 		ArmyTrailer = 2818520053u,
 		ArmyTrailer2 = 2657817814u,
+		Asbo = 1118611807u,
 		Asea = 2485144969u,
 		Asea2 = 2487343317u,
 		Asterope = 2391954683u,
+		Astron = 629969764u,
+		Astron2 = 2803699023u,
+		Autarch = 3981782132u,
 		Avarus = 2179174271u,
+		Avenger = 2176659152u,
+		Avenger2 = 408970549u,
+		Avisa = 2588363614u,
+		BF400 = 86520421u,
+		BJXL = 850565707u,
+		BType = 117401876u,
+		BType2 = 3463132580u,
+		BType3 = 3692679425u,
```

Comment:
These seems like re-additions, being removed from below and added here breaking the order